packages/dx-react-grid/docs/guides/virtual-scrolling.md

```diff
@@ -38,6 +38,16 @@ If the Grid should have the same size as the container element, set the `Virtual
 
 .embedded-demo({ "path": "grid-virtual-scrolling/stretching-to-parent-element", "showThemeSelector": true })
 
+## Scroll to Row
+
+To scroll the table to a particular row, call the `scrollToRow` method. It accepts a row ID. To call the method, you need the `VirtualTable` plugin's ref.
```

Comment:
Perhaps:

"To scroll the table to a particular row, call the `scrollToRow` method and pass the row ID as its parameter"?

Reply:
```suggestion
To scroll the table to a particular row, call the `scrollToRow` method and pass the row ID as its parameter. To call the method, you need the `VirtualTable` plugin's ref.
```